### ISSUE TITLE
Сутулые псы и помощь из зала.

### ISSUE THREAD

**Issue by Stritirok** [animusdev/green#127]
Это, ну у нас же тут серьезный ДЕВЕЛОПМЕНТ, так что да.

1. Собрать унитаз конечно очень весело, но собрать броню из металла как мне кажется веселее. В общем предлагаю некоторые штучки для крафта:

- Усиленная арматура - несколько арматуы и сварка - получаем эдакую палку боли. (что то такое - https://i.gyazo.com/ea96e51e9994f35de94def0e8c8e6bd7.png в общем вы знаете какой я спрайтер)
- Броня из металла - я знаю, что это ОЧЕНЬ оригинально и все дела, но почему нет?
Собирать ее наверное из металла с укрепметаллом, ну и инструменты по вкусу. В итоге получаем такую штуку на грудину, которая защищает от пулек и баллона, но не очень от лазера и всего остального. Так же можно сделать крафт щита, потому что щиты это круто.
(есть даже какое то подобие наработки - https://i.gyazo.com/14ada7743aca028c526e099b68b50dba.png)
- Возможно стоит сделать какой то крафт биты с гвоздями или чем нибудь еще(лол). Не самое важное, нужно и все такое, но мне кажется, что так будет забавно.
- Ножи или что то такое не вижу смысла впиливать, так как их предостаточно.
- Существует мнение, что подручные дробовики юзлес, так как их надо делать из модуляр-ресивера. Так что возможно стоит это переделать?
- Очень странная идея, но как насчет собрать плазма-ган, который стреляет мощно, но в руках может ебнуть? 

То бишь из моего словесного поноса думаю понятно, что стоит расширять возможности крафта, добавляя всякое-всякое, думаю идей у каждого есть туча насчет этого всего.

2. Слово про науку, наука это круто. Да. Так что думаю есть смысл ВЗЯТЬ все то, что есть в РНД новоТГ и засунуть к нам, это расширит наш эдакий ассортимент. 
Еще есть идея насчет своровать кое чего из РНД с ВГ-билда(например там достаточно неплохие есть пушечки, еще мне нравились новые штуки для медиков - лазерные скальпели и все такое), но вы сами знаете, что такое ВГ-билд.

3. Как вы поняли я поехавший на валынах дурачек, так что предлагаю респрайтнуть некоторые стволы, например Еган(вот например спрайт, что дала тэстимадам - https://i.gyazo.com/514274d550722bebc7e158c5341e44ad.png), револьверы синдикатеров, псевдо стечкин, риот-ган(извините, но даже тот, который был раньше нравился мне больше), впишите еще что то свое, если захотите.

4. Что то такое.

- Сделать квадроциклу анимацию езды(вращение колес) и какой то звук, а то слышать шаг при поездке с ветерком - странно.
- Вроде говорили, что собрать фармбота нельзя, если правда нельзя, то пора бы его уже починить.
- Уменьшить количество декларов для режимов, но это так.

Спасибо за внимание, всегда ваш сутулый пес.

**Comment by Stritirok** [comment]
Чет разметка сломалась.

**Comment by silentenemy** [comment]
Крафт веселый, но баланс надо бы.
> как насчет собрать плазма-ган, который стреляет мощно, но в руках может ебнуть? 

Advanced Energy Gun же.

**Comment by Stritirok** [comment]
> Advanced Energy Gun же.

Так это РнД, а я предлагаю прям из говна и палок, плюс там то просто хапнуть радиации, а тут тебе морду оторвет за то, что ты стрельнул слишком метко.

**Comment by silentenemy** [comment]
> Так это РнД, а я предлагаю прям из говна и палок, плюс там то просто хапнуть радиации, а тут тебе морду оторвет за то, что ты стрельнул слишком метко.

Мне нравится, удваиваю!

**Comment by Stritirok** [comment]
Добавлю, что могу если что намалять всякого, главное сказать что именно нужно нарисовать.

**Comment by bo20202** [comment]
мертворождённая тема с предложениями

**Comment by silentenemy** [comment]
@bo20202 ты чего, сейчас ещё РПГ-7 спиздим!

**Comment by Stritirok** [comment]
>мертворождённая тема с предложениями

Мы и так все мертвы внутри. Так что как раз.

**Comment by Dark-Magov** [comment]
> Мы и так все мертвы внутри. Так что как раз. 
Вот не надо так, я мёртв и снаружи.

**Comment by Stritirok** [comment]
>Мы и так все мертвы внутри. Так что как раз. 
>Вот не надо так, я мёртв и снаружи.

Тогда это тем более для тебя!

**Comment by Dark-Magov** [comment]
>Тогда это тем более для тебя!

Таки не соглашусь, мне больше хотет сидеть в незапущенном раунде, чем каэсочка.

**Comment by Stritirok** [comment]
>Таки не соглашусь, мне больше хотет сидеть в незапущенном раунде, чем каэсочка.

То бишь тебе не нравятся мои идеи?

**Comment by Dark-Magov** [comment]
А ещё твоя арматура похожа на булаву.

**Comment by Stritirok** [comment]
>А ещё твоя арматура похожа на булаву.

Я так и хотел.

**Comment by Dark-Magov** [comment]
> То бишь тебе не нравятся мои идеи? 

Нет, мне не нравятся пострелушки и нравится аутировать в незапущенном раунде.

**Comment by Stritirok** [comment]
>Нет, мне не нравятся пострелушки и нравится аутировать в незапущенном раунде.

У вас какие то серьезные проблемы.

**Comment by Stritirok** [comment]
Да и ясно дело, что ты будешь усиливать конец арматурки, чтоб больнее бить.

**Comment by Dark-Magov** [comment]
> Да и ясно дело, что ты будешь усиливать конец арматурки, чтоб больнее бить.

Почему сразу к огнетушителю арматуру концом не приварить?

**Comment by silentenemy** [comment]
> Почему сразу к огнетушителю арматуру концом не приварить?

Doubledamage

**Comment by Stritirok** [comment]
>Почему сразу к огнетушителю арматуру концом не приварить?
Потому что ты проецируешь.

**Comment by Stritirok** [comment]
Да и дай мне поиграть в геймдев, ты то уже наигрался со своими киборгами, теперь моя очередь.
